docs/micro-rdk/board/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Micro-RDK Board Models"
+linkTitle: "Board"
+weight: 30
+type: "docs"
+description: "The micro-RDK support for the signal wire hub of a robot, with GPIO pins for transmitting signals between the robot's computer and its other components."
+images: ["/icons/components/board.svg"]
+tags: ["board", "components", "micro-rdk"]
+no_list: true
+# SMEs: Nick M., Gautham V.
+---
+
```

Comment:
[opt] If you included a bit more info here about what a board is, then all the places where you link to "board" in micro-RDK-specific-contexts (i.e. attribute tables as discussed earlier) could link here instead of to the RDK one, which could be less confusing to users.

Reply:
Done!